static/src/javascripts/projects/common/modules/commercial/dfp/breakout-iframe.js

```diff
@@ -45,16 +44,11 @@ define([
             $breakoutEl = $('.breakout__html, .breakout__script', iFrameBody);
 
             if ($breakoutEl.hasClass('breakout__html')) {
-                fastdom.write(function () {
+                resolve(fastdom.write(function () {
                     $iFrame.hide();
                     $breakoutEl.detach();
-                    $iFrameParent.append($breakoutEl[0].children);
-                }).then(function () {
-                    var $responsiveAds = $('.ad--responsive', $iFrameParent[0]);
-                    resolve(fastdom.write(function () {
-                        $responsiveAds.addClass('ad--responsive--open');
-                    }));
-                });
+                    $slot.append($breakoutEl[0].innerHTML);
```

Comment:
Using `.children` instead of `.innerHTML` will save the browser a whole lot of work.